nixos/modules/services/continuous-integration/buildbot/master.nix

```diff
@@ -92,10 +92,7 @@ in {
       workers = mkOption {
         type = types.listOf types.str;
         description = "List of Workers.";
-        default = [
-          "worker.Worker('example-worker', 'pass')"
-        ];
-        example = [ "worker.LocalWorker('example-worker')" ];
```

Comment:
i thought it was redundant, as default serves as an example.  This follows the pattern with other vars in which this is the case, such as "home", "user" and "port".